docs/contributing/how-to-contribute.md

```diff
@@ -78,11 +78,12 @@ The core team is monitoring for pull requests. We will review your pull request
 
 1. Fork [the repository](https://github.com/facebook/react) and create your branch from `master`.
 2. If you've added code that should be tested, add tests!
-3. If you've changed APIs, update the documentation.
-4. Ensure the test suite passes (`npm test`).
-5. Make sure your code lints (`npm run lint`).
-6. Run the [Flow](https://flowtype.org/) typechecks (`npm run flow`).
-7. If you haven't already, complete the CLA.
+3. If you added or removed any tests, run ./scripts/fiber/record-tests before submitting the pull request, and commit the resulting changes.
```

Comment:
Also, this step should probably come the last before CLA. It should be done after everything else is ready.